docs/specviz/notebook.rst

```diff
@@ -1,5 +1,19 @@
 ***********************************
-Using SpecViz in a Jupyter Notebook 
+Using Specviz in a Jupyter Notebook 
 ***********************************
 
-Words go here...
+Specviz is developed to fully support analyzing spectra within your existing `Jupyter notebooks <https://jupyter.org/>`_! To use Specviz, install jdaviz in your notebook's python environment and add a new cell wherever you would like to use Specviz.
+
+    >>> # Import specviz
+    >>> from jdaviz import SpecViz
+    >>> # Instantiate an instance of Specviz
+    >>> specviz = SpecViz()
+    >>> # Display Specviz
+    >>> specviz.app
```

Comment:
```suggestion
    >>> specviz.app #doctest: +SKIP
```

This will skip running this line in the doctests and presumably fix the CI errors